randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        ),
+    ],
+    "2410883438": [
+        # Just Hold L Tip
+        # Replaces Main Plaza bird image scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; grants greater "
+            "control over aerial movement, making precise jumps easier."
+        ),
+    ],
+    "389886815": [
+        # Just Tap B Tip
+        # Replaces Observatory left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When performing a scan dash or lock-on dash, jump by tapping "
+            "&image=SI,0.70,0.68,833BEE04; for as little time as possible to gain extra height."
+        ),
+    ],
+    "3668253850": [
+        # Item Drop Tip
+        # Replaces Waterfall Cavern hole scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Each room has different pickups that drop from crates and defeated enemies.\n\n"
+            "If a specific pickup does not seem to be appearing (e.g. Power Bomb ammo), try a different room."
+        ),
+    ],
+    "3673769160": [
+        # L-Jump Frames Tip
+        # Replaces Elite Research top floor computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; for the first 13 frames of a jump "
+            "and then letting go\nslightly extends jumps. This is called an L-Jump. When using the &main-color=#FF3333;"
+            "Space Jump Boots&main-color=#89D6FF;, holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; "
+            "for 7 frames after the second jump grants an additional, smaller extension."
+        ),
+    ],
+    "1807281950": [
+        # RDV FAQ Tip
+        # Replaces North Atrium right sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;Each game in Randovania has a Frequently Asked Questions section.",
+    ],
+    "3979124144": [
+        # Spoiler Log Tip
+        # Replaces North Atrium left sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;By default, seeds in Randovania generate with spoiler logs.\n\n"
+            "Spoiler logs can be used to check what items are at specific locations, among other information."
+        ),
+    ],
+    "2367804388": [
+        # Shiny Missile Tip
+        # Replaces North Atrium disc scans
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;There is a 1/1024 chance for Missile Expansions to be &main-color=#FF3333;shiny"
+            "&main-color=#89D6FF;.\n\n\nShiny Missile Expansions are "
+            "&main-color=#3333FF;blue&main-color=#89D6FF;.\n\n\n\nThat is all."
```

Comment:
Same as above

Reply:
Goal 2, and again, I think preempting "what is this?" questions + a fun fact that isn't competing for real estate is worth keeping.